clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/mark_node_failed

```diff
@@ -15,14 +15,19 @@ etcd_key=clearwater
 
 . /usr/share/clearwater/utils/check-root-permissions 1
 
-if [ $# -ne 3 ]
+if [ $# -ne 3 ] && [ $# -ne 4 ]
 then
   echo "Usage: mark_node_failed [node_type] [datastore] [failed_node_ip]"
+  echo "       mark_node_failed [node_type] [datastore] [failed_node_ip] [etcd_ip]"
```

Comment:
Can you change this so that the non optional parameters are surrounded by <>, and the optional one by [], and then only have the one usage?